tests/Unit/ColumnTest.php

```diff
@@ -136,6 +137,52 @@ public function testColumnWithClosures()
         $this->assertSame('BAR', $column->transform(null));
     }
 
+    public function testMoneyColumn()
+    {
+        $column = new MoneyColumn();
+        //Test with defaults
+        $column->initialize('test', 1, [], new DataTable($this->createMock(EventDispatcher::class)));
```

Comment:
Please use `$this->createDataTable()`, it's the future-proof provider of this mock.

Reply:
Changed